code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -592,17 +592,13 @@
 		mode()
 
 /mob/living/simple_animal/swap_hand(hand_index)
+	. = ..()
```

Comment:
For some reason this concerns me.

Reply:
That we follow the chain down to the base proc? Or that this is using proper signals?

Reply:
Issue resolved in discord, This is needed because the base proc sends a swap hands signal that can be blocked.